ui/client/components/graph/Graph.js

```diff
@@ -152,39 +167,23 @@ class Graph extends React.Component {
     return this.state.exported
   }
 
-  validateConnection = (cellViewS, magnetS, cellViewT, magnetT) => {
-    const from = cellViewS.model.id
-    const to = cellViewT.model.id
-    return magnetT && NodeUtils.canMakeLink(from, to, this.props.processToDisplay, this.props.processDefinitionData)
+  validateConnection() {
```

Comment:
why is it better than previous version? I'm just curious ;)

Reply:
I try to separate component props from inner joint/paper args